CoreScriptsRoot/Modules/VR/Panel3D.lua

```diff
@@ -628,32 +713,194 @@ end
 
 --Child class, Subpanel
 local Subpanel = {}
-local Subpanel_mt = {}
-function Subpanel.new(guiElement)
-	local instance = {
-		guiElement = guiElement
-	}
-	return setmetatable(instance, Subpanel_mt)
+local Subpanel_mt = { __index = Subpanel }
+function Subpanel.new(parentPanel, guiElement)
+	local self = setmetatable({}, Subpanel_mt)
+	self.parentPanel = parentPanel
+	self.guiElement = guiElement
+	self.lastParent = guiElement.Parent
+	self.ancestryConn = nil
+	self.changedConn = nil
+
+	self.lookAtPixel = Vector2.new(0,0)
+	self.lookedAt = fals
```

Comment:
Of course. It's missing "local fals = false". 

On a more serious note, nil evaluates to false when coerced to boolean... so functionally speaking it isn't wrong...